shell/platform/darwin/common/framework/Headers/FlutterTexture.h

```diff
@@ -26,9 +32,26 @@ FLUTTER_EXPORT
 @end
 
 FLUTTER_EXPORT
+/**
+ * A collection of registered `FlutterTexture`'s.
+ */
 @protocol FlutterTextureRegistry <NSObject>
+/**
+ * Registers a `FlutterTexture` for usage in Flutter and returns an id that can be used to reference
+ * that texture when calling into Flutter with channels.
+ */
 - (int64_t)registerTexture:(NSObject<FlutterTexture>*)texture;
+/**
+ * Notifies Flutter that the content of the previously registered texture has been updated.
+ *
+ * See also: `flutter::PlatformView::MarkTextureFrameAvailable`
```

Comment:
This is an internal method that users of this header will not have access to.

Reply:
Maybe also mention that in response to this call, the engine will attempt to resolve the pixel buffer via the `copyPixelBuffer` call on the render thread.

Reply:
Render thread?  Gpu thread, right?
Also, this call is meant to happen on the platform thread, right?

Reply:
Updated.

Reply:
This call should happen on the platform thread but the pixel buffer will be copied on the GPU thread. Sorry about the mixed terminology, still processing https://github.com/flutter/flutter/issues/29443.